api/opds.py

```diff
@@ -350,16 +349,11 @@ def related_books_available(cls, work, library):
         """:return: bool asserting whether related books might exist for
         a particular Work
         """
-        if isinstance(work, Work):
-            edition = work.presentation_edition
-        else:
-            # This is a MaterializedWork*, so we're gonna grab the
-            # edition where we can.
-            edition = work.license_pool.presentation_edition
+        contributions = work.sort_author and work.sort_author != Edition.UNKNOWN_AUTHOR
```

Comment:
We don't need to look at the contributions (which would cause a database hit) here, we just need to see whether, when we calculated the materialized work's sort_author, we ended up with a non-default value.